scripts/download.py

```diff
@@ -0,0 +1,174 @@
+import arcpy
+import os
+import csv
+from glob import glob
+from shutil import rmtree
+import itertools
+import json
+
+
+# subCategoryText = {
```

Comment:
Can this block of commented code be removed? Or maybe it's documenting something?